src/Documentation/Advanced-Concepts/StatelessWorker.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: page
+title: Stateless Worker Grains
+---
+
+# Stateless Worker Grains
+
+By default, the Orleans runtime creates no more than one activation of a grain within the cluster.
+This this the most intuitive expression of the Virtual Actor model with each grain corresponding to an entity with a unique type/identity.
+However, there are also cases when an application needs to perform functional stateless operations that are not tied to a particular entity in the system.
+For example, if client sends requests with encrypted payloads that need to be decrypted before they could be routed to the target grain for processing, such decryption/routing logic is not tied to a specific entity in the application, and can easily scale out.
```

Comment:
a better example I think is compression/uncompression, since encryption has some state - a key (and thus may raise other unrelated open questions), while uncompression is fully stateless.